frontend/src/components/form/ColorPicker.vue

```diff
@@ -0,0 +1,76 @@
+<!--
+Displays a field as a color picker (can be used with v-model)
+-->
+
+<template>
+  <div>
+    <v-menu
+      ref="menu"
+      v-model="showColorPicker"
+      :close-on-content-click="false"
+      transition="scale-transition"
+      offset-y
```

Comment:
```suggestion
      offset-y
      offset-overflow
```
Damit es nicht das Textfeld verdeckt.

Reply:
Hab irgendwie noch nicht verstanden, was das macht.

Reply:
Damit das Textfeld nicht verdeckt wird. Dazu wird das Menu standardmässig unterhalb gezeigt, falls aber nicht genügend Platz unterhalb des Feldes ist, wird es oberhalb davon angezeigt.

Reply:
Ok, done